openfe/tests/helpers.py

```diff
@@ -0,0 +1,194 @@
+from itertools import chain
+
+import numpy as np
+import openmm
+from gufe import LigandAtomMapping, ProteinComponent, SolventComponent
+from openff.units.openmm import ensure_quantity, from_openmm, to_openmm
+from openmm import app, unit
+from openmmforcefields.generators import SystemGenerator
+
+from openfe.protocols.openmm_rfe import _rfe_utils
+from openfe.protocols.openmm_rfe._rfe_utils.relative import HybridTopologyFactory
+from openfe.protocols.openmm_utils import system_creation
+
+
+def make_htf(
+    mapping: LigandAtomMapping,
+    settings,
+    protein: ProteinComponent = None,
+    solvent: SolventComponent = None,
+) -> HybridTopologyFactory:
+    """Code copied from the RBFE protocol to make an HTF."""
+
+    system_generator = SystemGenerator(
```

Comment:
Rather than doing all this, and being at risk of dealing with flakyness with variabilities in system building, why not just serialize the two end state systems? This is the approach openmmtools does for factory testing, so do we internally for some of the absolute factory things, it would make life a ton easier.